tests/test_dataclass_reader.py

```diff
@@ -88,3 +89,29 @@ def test_csv_header_items_with_spaces_together_with_skipinitialspaces(create_csv
 
         assert user.name == 'User1'
         assert user.age == 40
+
+
+def test_parse_bool_value(create_csv):
+    for true_value in ['yes', 'true', 't', 'y', '1']:
+        csv_file = create_csv({'name': 'User1', 'age': 40, 'boolValue': '{}'.format(true_value)})
```

Comment:
Since the `DataclassWithBooleanValue` has only one field, it is not necessary to create csv with name and age.

Also, instead of doing: 
```python
{'boolValue': '{}'.format(true_value)}
```
Can use f-strings:
```python
{'boolValue': f'{true_value}'}
```

Reply:
Good catch, and thanks for the tip on f-strings.